lib/utils/log.js

```diff
@@ -52,7 +52,8 @@ log.debug = function ({ type='', pre='', data='', post='' }) {
 };
 
 log.add_to_debug_log = function(value) {
-  fs.appendFileSync(log.debug_log_file, value);
+  let artifacts_path = session_vars.get_artifacts_path_name();
+  fs.appendFileSync(`${ artifacts_path }/${ log.debug_log_file}`, value);
```

Comment:
Hmm, have we tested this out? Was there a reason we weren't putting this stuff in that folder? Or was it just being added to that folder somewhere else? It's weird that things were functioning when that line was missing.

Reply:
Decided to just remove it; it's not being used anywhere / is probably currently broken, and this was a guess at how to fix to, but didn't start using the function anywhere.

Reply:
Issue #659 as it's currently not being used anywhere.